server/index.js

```diff
@@ -20,13 +20,44 @@ const main = async () => {
     database: 'todo-app',
   })
   await client.connect()
+  
+  //Retrieve all tasks in a week based on the start and end date
+  app.get('/', async (req, res) => {
+    const response = JSON.stringify(
+      (
+        await client.query(sql`
+  SELECT *
+  FROM "task"
+  WHERE "task.dueDate" >= req.query.start AND
+  "task.dueDate" <= req.query.end
+  `)
+      ).rows,
+    )
+    res.end(response)
+  })
+
+  //retrieve task based on ID
+  app.get('/', async (req, res) => {
```

Comment:
We can make it so that the URL is like `/tasks/20` if you change the URL parameter to `/tasks/:id`. Then you can access the ID inside the function using `req.params.id` ([example](https://github.com/lukeed/polka#usage))